### PR TITLE
Add features: gradio app & local chatbot support

### PR DESCRIPTION
**Pull request by BinjieW**
* host gradio app on hugging face space
* support local chatbot to replace gpt via fastchat RESTful API server

### CODE REVIEW

**Comment by EthanC111** [comment]
LGTM!

File: app.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      model=model,
+      messages=[
+            {"role": "system", "content": "You are a helpful assistant."},
+            {"role": "user", "content": message},
+        ]
+    )
+    return response.choices[0].message['content']
+
+def fact_check(openai_api_key, serper_api_key, scraper_api_key, model, message, response, category):
+    os.environ['SCRAPER_API_KEY'] = ''
```
Comment (pfliu-nlp) [inline app.py]: This type of implementation is kinda strange. Let's create an issue for future improvement.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+openai==0.27.8
```
Comment (pfliu-nlp) [inline requirements.txt]: We don't need to have this file please delete it (since we have claimed the dependencies in setup.cfg.

Comment (BinjieW) [reply requirements.txt]: ok. This file is for hugging face space, and I will add it to .gitignore

File: app.py (hunk 1 above)
Comment (BinjieW) [reply app.py]: ok, I will open an issue.

**Pull request merged**